### DISCUSSION TITLE
Calculation of statistical inefficiency in pymbar

### DISCUSSION THREAD

**Discussion by matspunt** [choderalab/pymbar#491] [Q&A]
Hi, 

I have a question about implementation of the autocorrelation time functions and determination of the statistical inefficiency in the _pymbar.timeseries_ module. I could not figure out the exact observable that is used for calculation of the autocorrelation function. I.e. in the documentation it is stated that: 

"_Notes: The autocorrelation of the timeseries is used to compute the statistical inefficiency. The normalized fluctuation
autocorrelation function is computed by averaging the unnormalized raw correlation functions. The fast method
described in Ref [1] is used to compute g_"

Equations 15-21 in _ref 1_ demonstrate the general principle of approaching a decorrelation but what observable is used in practice in pymbar? The actual free energy differences between individual/block averaged frames, the interaction levels of LJ/Coulomb interactions (in alchemical transformations, some other thing I am not aware of? Perhaps I am misunderstanding but this was not immediately clear to me from the documentation, I appreciate any guidance. 

Thanks a lot. 

1. J. D. Chodera, W. C. Swope, J. W. Pitera, C. Seok, and K. A. Dill. Use of the weighted histogram analysis method for the analysis of simulated and parallel tempering simulations. JCTC 3(1):26- 41, 2007

**Comment by mrshirts** [comment]
You choose - it's a function you can pass a time series of whatever variable you feel is most important.  The RIGHT answer is not always obvious.

**Reply by matspunt** [reply]
Really appreciate the reply, thanks a lot. I had hoped there was some magic 'use this <observable> and it will work for you in each context' but such magic bullets rarely seem to exist in FE calculations I am increasingly learning to appreciate :). Thanks again!
